payments/index.md

```diff
@@ -1,8 +1,209 @@
+# The Fundamentals
+
+>eCommerce APIs share a common foundation with a similar payment process for all payment methods, which reduce complexity and enable a more straightforward integration.
+
+*Two-phase payments* ([Card](/payments/credit-card), [Invoice](/payments/invoice), [MobilePay](/payments/invoice), [Vipps](/payments/vipps/))
+
+A two-phase payment is managed in two steps - first reserving funds that will be captured in the next step. The most common payment instrument using two-phase payments is card payments.
+
++ [Creating a payment object](#creating-a-payment-object).  
+The payment is the container object that holds all transactions that will be created during the payment process.  When PayEx receives the payment request body (in JSON format), a payment  is created and you will be given a unique paymentID in return. The response also include  (in a true RESTful way) the URIs to possible further actions, given the state of the payment.
++ Authorize funds.  
+  An authorization transaction reserves the funds necessary funds . It is possible to abort a payment before the end user has fulfilled the payment process.  
+And either:  
++ Capture funds.  
+  Before delivering the merchandise you need to create a capture transaction to ensure that the money is charged from the consumer credit card or properly billed by invoice.  
+Or:  
++ Cancel the authorized amount.  
+ Funds that are authorized but not yet captured can be released back to the consumer. This is done by creating a cancel transaction. 
++ Reverse captured funds.  
+ In some cases you may need to make a reversal of captured funds. This is achieved by creating a reversal transaction.
+
+
+*One-phase payments* ([Direct Debit](/payments/direct-debit), [Swish](/payments/swish), [Card](/payments/credit-card))
+
+If you use a one-phase method (like Direct Bank Debit or [Swish](/payments/swish)) a Sales transaction will be created and the consumer charged right away. 
+
+* _One-phase card payments is enabled with autocapture._
+
+>Add image
+
+### <a name="creating-a-payment-object">Creating a payment object</a>.
+The `payment` is the container object that holds all transactions that will be created during the payment process. When PayEx receives the payment request body (in JSON format), a payment  is created and you will be given a unique paymentID in return. The response also include  (in a true RESTful way) the URIs to possible further actions, given the state of the payment. It is possible to abort a payment before the end user has fulfilled the payment process.
+Settle funds directly with a Sales transaction.  
+```
+Pick and choose between the payment methods best suited for your business. Take advantage of our easy-to-use PCI compliant platforms Redirect and PayEx Hosted View - or use PayEx Direct API to integrate directly. Our payment methods and their platform availability are listed in the table below.
```

Comment:
This looks more like prose than code to me, so wrapping it in code fences does not feel right. A quote `>` might be more suitable?